<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="input_id">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PW</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_pw">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>로그인</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4H</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_member">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>회원가입</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_login">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>93</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>로그인</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>회원이 아니신가요?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
